.env.gold

```
@@ -21,3 +21,8 @@ NEXT_PUBLIC_THEME_SIDEBAR_ACCENT_FOREGROUND_LIGHT="22.7273 82.5% 31.3725%"
 NEXT_PUBLIC_THEME_SIDEBAR_ACCENT_LIGHT="48 100% 96.0784%"
 NEXT_PUBLIC_THEME_STABLE_TOKEN="#ffd007"
 NEXT_PUBLIC_THEME_VOLATILE_TOKEN="#e3201f"
+
+NEXT_PUBLIC_NEUTRON_ID=886b7721bef42f60c6317d37d8752da8aca01898cae7dae61808c4a14225edc8
+NEXT_PUBLIC_PROTON_ID=9944ff273ff169f32b851b96bbecdbb67f223101c15ae143de82b3e7f75b19d2
+NEXT_PUBLIC_GLUON_NFT=797e331df22c5cfd0aae654703ebc12dcbcba99b80d32e2b7e151eac2d27b6fa
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GLUON_NFT key should go before the NEXT_PUBLIC_NEUTRON_ID key

(UnorderedKey)